### ISSUE TITLE
Computed bindings don't run when an argument is undefined

### ISSUE THREAD

**Issue by stephenbunch** [Polymer/polymer#3401]
Hey guys,

I'm trying to use a computed binding where one of the input arguments is `undefined`. The computed binding is never executed. However, it runs as expected when there is only one argument.

http://codepen.io/stephenbunch/pen/XXoWbe

**Comment by MeTaNoV** [comment]
This is normal behavior, check that part of the doc: https://www.polymer-project.org/1.0/docs/devguide/data-binding.html#dependent-properties

**Comment by stephenbunch** [comment]
@MeTaNoV Ahh, I see.

> The computing function **is not called until all dependent properties are defined (!=undefined)**.

In that case, it still doesn't make sense that it works when there's only one argument.

**Comment by MeTaNoV** [comment]
you're right, it shouldn't!

**Comment by MeTaNoV** [comment]
I did some spelunking and found out that in https://github.com/Polymer/polymer/blob/master/src/lib/bind/effects.html#L123, you exit like:
```
        if (args.length > 1 && v === undefined) {
          return;
        }
```
where I assumed that we would escape at the first undefined argument? 
Therefore changing to 
```
        if (args.length > 0 && v === undefined) {
          return;
        }
```
solve this issue but might introduce other unwanted effect?
@azakus WDYT?
(I tried running `wct` on master and with or w/o the modification, I ended up with 357 tests failed on Chrome... oO)
